frontend/src/components/Menu/Menu.js

```diff
@@ -1,12 +1,23 @@
 import "./Menu.scss";
 import React from "react";
+import ky from "ky";
 import { Button } from "@material-ui/core";
 import { useHistory } from "react-router";
 import { buttonSound } from "../shared/Sounds";
 
 const Menu = () => {
   const history = useHistory();
 
+  const logout = async () => {
+    try {
+      await ky.get("/api/logout");
+      history.push("/login");
+      window.location.reload();
```

Comment:
Could you try not to user `reload`?

Reply:
in this approach of checking login I don't know how to omit this